packages/core/src/modules/credentials/protocol/revocation-notification/services/RevocationNotificationService.ts

```diff
@@ -116,14 +143,15 @@ export class RevocationNotificationService {
 
     const credentialId = messageContext.message.credentialId
 
-    if (messageContext.message.revocationFormat !== v2IndyRevocationFormat) {
+    if (![v2IndyRevocationFormat, v2AnonCredsRevocationFormat].includes(messageContext.message.revocationFormat)) {
       throw new AriesFrameworkError(
-        `Unknown revocation format: ${messageContext.message.revocationFormat}. Supported formats are indy-anoncreds`
+        `Unknown revocation format: ${messageContext.message.revocationFormat}. Supported formats are indy-anoncreds and anoncreds`
```

Comment:
Is the RFC extended with this new format?

Reply:
RFC has been updated